src/transformers/models/marian/modeling_marian.py

```diff
@@ -477,7 +477,7 @@ class MarianPreTrainedModel(PreTrainedModel):
     base_model_prefix = "model"
     supports_gradient_checkpointing = True
 
-    def _init_weights(self, module):
+    def _init_weights(self, module: Union[nn.Linear, nn.Embedding, MarianSinusoidalPositionalEmbedding]):
```

Comment:
A lesser contributor would have just given up and written `Any` for this one. I respect it.